documentation/src/docs/deployment/manual-install.md

```diff
@@ -148,8 +151,11 @@ autoScale.controllerUri=tcp://<controller-ip>:9090
 hdfs.hdfsUrl=<hdfs-ip>:8020
 ```
 
-Once the configuration changes have been made you can start the segment store with:
+After making the configuration changes, the segment store can be run using the following command:
 
 ```
 bin/pravega-segmentstore
 ```
+## Runing Pravega Controller and Segment Store with security enabled
+Here are the details for configurations for [Pravega controller](../security/pravega-security-configurations.md#Pravega controller) and [Pravega Segment store](../security/pravega-security-configurations.md#Pravega Segmentstore).
+These parameters can be changed to represent required security configurations.
```

Comment:
@arvindkandhare  Please provide the links

Reply:
@fpj @arvindkandhare 
The following links are not enabled.

[Pravega controller](../security/pravega-security-configurations.md#Pravega controller) and [Pravega Segment store](../security/pravega-security-configurations.md#Pravega Segmentstore). These parameters can be changed to represent required security configurations.

Reply:
@Aparnarr I don't understand the ask here. If they aren't enabled, can we enable them? These seem to be pointing to section anchors, so it is just a matter of adding the anchors, right?